nixos/modules/hardware/gpu/intel-gpu-tools.nix

```diff
@@ -0,0 +1,27 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
```suggestion
```

top-level `with lib;` is discouraged, let's avoid this.

Reply:
This is something I've always been pretty conflicted on. It's discouraged in general, but like the common use of `with lib` for `meta`, almost all (~80%) nixos modules follow this pattern and it seems more out of place to not do this.

I'm ok with changing this though!